virttest/qemu_monitor.py

```diff
@@ -2506,3 +2506,21 @@ def drive_backup(self, device, target, format, sync, speed=0,
         if speed:
             args["speed"] = speed
         return self.cmd(cmd, args)
+
+    def press_down_key(self, key):
+        """
+        Press down the key to VM
+        param key: the key to press
+        :return: Command output
+        """
+        cmd = "input-send-event"
+        self.verify_supported_cmd(cmd)
+        args = {"events": [{
+            "type": "key",
+            "data": {
+                "down": True,
```

Comment:
Or we can add a new param : 
1. "down": True  hold the key on
2. '"down": False  release the key